docker/superset-omnisci.py

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
It seems that superset uses pyarrow 4 (https://github.com/apache/superset/blob/master/setup.py#L103) and pyomnisci uses pyarrow 3 (https://github.com/omnisci/pyomnisci/blob/master/setup.py#L13) , so maybe it would create some conflicts